gips/utils.py

```diff
@@ -389,15 +389,27 @@ def mosaic(images, outfile, vector):
         filenames.append(images[f].Filename())
     # transform vector to image projection
     geom = wktloads(transform_shape(vector.WKT(), vector.Projection(), srs))
-
     extent = geom.bounds
-    ullr = "%f %f %f %f" % (extent[0], extent[3], extent[2], extent[1])
-
-    # run merge command
+    # part of the command string
     nodatastr = '-n %s -a_nodata %s -init %s' % (nd, nd, nd)
-    cmd = 'gdal_merge.py -o %s -ul_lr %s %s %s' % (outfile, ullr, nodatastr, " ".join(filenames))
-    result = commands.getstatusoutput(cmd)
-    VerboseOut('%s: %s' % (cmd, result), 4)
+    # might reuse the gdal merge command
+    def gdal_merge(x0, y0, x1, y1):
+        ullr = "%f %f %f %f" % (x0, y0, x1, y1)
+        cmd = 'gdal_merge.py -o %s -ul_lr %s %s %s' % (outfile, ullr, nodatastr, " ".join(filenames))
+        result = commands.getstatusoutput(cmd)
+        VerboseOut('%s: %s' % (cmd, result), 4)
+        return result[0] == 0
```

Comment:
more optional tidying, only if now's the right time:  `VerboseOut` is being transitioned to `verbose_out`.  New code should avoid `"%f %s" % (n, s)` in favor of `"{} {}".format(n, s)`.  I like `return result[0] == 0` though, nicely brief and clear.  : ]